inc/class-wpseo-admin-bar-menu.php

```diff
@@ -546,11 +582,11 @@ protected function get_post_score( $post ) {
 	/**
 	 * Gets the current term if in a singular term context.
 	 *
-	 * @global string       $pagenow  Current page identifier.
+	 * @return WP_Term|null Term object, or null if not in singular context.
 	 * @global WP_Query     $wp_query Current query object.
 	 * @global WP_Term|null $tag      Current term object, or null if none available.
 	 *
-	 * @return WP_Term|null Term object, or null if not in singular context.
+	 * @global string       $pagenow  Current page identifier.
```

Comment:
Same as above: As far as I know, `@global` notations should come before and `@return` be the last one.

Reply:
Fixed